docs/index.md

```diff
@@ -219,11 +219,14 @@ your `.pre-commit-config.yaml` file like so:
     rev:  # Fill in a tag / sha here
     hooks:
     -   id: gitlint
-        stages: [commit-msg]
```

Comment:
Yes if the upstream repo defines the `stages` then the user does not have to, if he/she does not wish to change them.

Reply:
Is `stages` not required? Does it get taken automatically from the upstream repo's `.pre-commit-hooks.yaml`?